x-pack/plugins/security_solution/cypress/cypress.json

```diff
@@ -1,6 +1,7 @@
 {
   "baseUrl": "http://localhost:5601",
   "defaultCommandTimeout": 120000,
+  "experimentalNetworkStubbing": true,
```

Comment:
See here: https://docs.cypress.io/api/commands/route2.html#Comparison-to-cy-route

I'm using an "experimental feature", `route2`, which allows for dynamic stubbing. I need this in order to stub different responses depending on request params